.github/workflows/markdown-lint.yml

```diff
@@ -0,0 +1,16 @@
+name: Lint Markdown files
+
+on: ["push", "pull_request"]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Lint Markdown files
+      uses: avto-dev/markdown-lint@v1
+      with:
+        config: '.github/workflows/config/markdown-lint.json'
```

Comment:
Could we move this configuration file to the repository root level, rename it `.markdownlint.json`, then update this YAML to point to the new path?

By doing so, local development tools which use some flavor of [markdownlint](https://github.com/DavidAnson/markdownlint) should automatically pick up the configuration, so that the experience is the same locally and in CI. I tested it out by copying the contents of this JSON file into a new root-level `.markdownlint.json` file and confirmed the `MD041` rule gets ignored when I move the file there, without needing to make any configuration changes in my IDE.

More information about the default config behavior here: https://github.com/DavidAnson/markdownlint#config